provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -68,7 +68,8 @@ def _create_node_descriptions(self) -> List[NodeDesc]:
         for machine_id in machines.keys():
             node_type = conf.get(f'node>{machine_id}>type')
             # Skipping for controller node
-            if node_type == 'storage_node':
+            if (node_type in
```

Comment:
node_type is decided to be dynamic and is subject to change in future, provisioner has publish a list of constant keys,
at https://github.com/Seagate/cortx-utils/pull/640/files.
We can use `SERVICE_MOTR_IO` macro to search for io services, thus, if ioservices are configured on a node then execute mkfs else not. We need to use conf-store's search api as below,
e.g.

Conf.search(self.default_index, parent_key, key_to_search, value_to_search)